src/ucp/core/ucp_context.c

```diff
@@ -545,8 +571,10 @@ static ucs_status_t ucp_add_tl_resources(ucp_context_h context,
                                          unsigned *num_resources_p,
                                          ucs_string_set_t avail_devices[],
                                          ucs_string_set_t *avail_tls,
+                                         ucs_string_set_t *avail_mem_tls,
```

Comment:
q/minor: maybe `avail_memtype_tls` to be consistent w/ `memtype_tls` in UCP config?